blocks/header/header.js

```diff
@@ -192,85 +161,74 @@ function decorateMiddleNav(nav) {
   a.setAttribute('aria-label', 'Sunstar Home');
 }
 
-/* Decorate the other items - which is the items pulled from top nav */
-function decorateOtherItems(otherItemsEl) {
-  otherItemsEl.classList.add('other-items');
-
-  /* Pull items from the top nav */
-  document.querySelector('nav.nav-top').querySelectorAll(':scope>ul>li').forEach((li) => {
-    otherItemsEl.appendChild(li.cloneNode(true));
-  });
-
-  /* Make a lang picker for mobile */
-  const langPicker = document.createElement('li');
-  langPicker.classList.add('mobile-lang-picker');
-  const langPickerUl = document.createElement('ul');
-  langPicker.appendChild(langPickerUl);
-  otherItemsEl.querySelector('.lang-picker').querySelectorAll(':scope>ul>li').forEach((li) => {
-    langPickerUl.appendChild(li.cloneNode(true));
-  });
-
-  langPicker.querySelectorAll(':scope>ul>li').forEach((li) => {
-    li.classList.add('mobile-lang-picker-item');
-    li.classList.remove('lang-picker-item');
+function getNavbarToggler() {
+  const navbarToggl = htmlToElement(`<button class="navbar-toggler">
+  <span class="mobile-icon">
+    <i></i>
+    <i></i>
+    <i></i>
+    <i></i>
+  </span>
+  </button>`);
+  const widerScreenWidth = window.matchMedia('(min-width: 77rem)');
+  if (!widerScreenWidth.matches) {
+    navbarToggl.classList.add('visible');
+  }
+  navbarToggl.addEventListener('click', () => {
+    const navBottom = document.querySelector('.nav-bottom');
+    const header = document.querySelector('header');
+    const { body } = document;
+    if (navBottom.classList.contains('open')) {
+      navBottom.classList.remove('open');
+      header.classList.remove('menu-open');
+      body.classList.remove('fixed');
+    } else {
+      navBottom.classList.add('open');
+      header.classList.add('menu-open');
+      body.classList.add('fixed');
+    }
   });
-
-  otherItemsEl.querySelector('.lang-picker').replaceWith(langPicker);
+  return navbarToggl;
 }
 
-function decorateBottomNav(nav, placeholders) {
-  addLevels(nav);
-  nav.querySelectorAll(':scope .menu-level-1-item').forEach((li) => {
-    buildDropDownMenu(li, placeholders);
-  });
+function attachWindowResizeListeners(nav) {
+  const header = document.querySelector('header');
+  const { body } = document;
+  const widerScreenWidth = window.matchMedia('(min-width: 77rem)');
+  widerScreenWidth.addEventListener('change', (event) => {
+    const toggler = nav.querySelector('.navbar-toggler');
+    if (event.matches) {
+      if (nav.classList.contains('open')) {
+        nav.classList.remove('open');
+        header.classList.remove('menu-open');
+        body.classList.remove('fixed');
+      }
+      if (toggler.classList.contains('visible')) {
+        toggler.classList.remove('visible');
+      }
```

Comment:
Nit:  How about `element.toggle([class]);` ?

Reply:
while opening level-2 items on the mobile view we are already removing visible class, so no need to remove or add it again at the time of resizing of window that is why toggle would be buggy here

Reply:
😕